New situations recently have introduced these ideas into the style guide (#9894)
gatsbyjs/gatsby: docs/docs/gatsby-style-guide.md

for a grade level of 6. Another tool available is the De-Jargonizer, originally
designed for scientific communication but might help avoid overspecialized
wording.

## Best practices
### Support software versions

When Gatsby commits to support a specific version of software (e.g. Node 6 and up), this is reflected in documentation. Gatsby documentation should be usable by all people on supported software, which means we don't introduce any commands or practices that can't be used by people on versions we've committed to support. In rare circumstances, we'll consider mentioning a newly introduced commands and practices as side notes.

> For example, npm 5.2.0 (which includes Node 8) introduced a command called `npx` that is not available for versions of Node below 8. Since Gatsby supports Node 6 and up, documentation should only introduce `npx` as an optional command in a note like so:

Reviewer: > npm 5.2.0 (which includes Node 8)
`npm` gets installed when you install `Node` (not the other way around), so I think this would need to be re-worded a bit to be factually correct.

> > npm 5.2.0 (which includes Node 8) introduced a command called `npx` that is not available for versions of Node below 8. Since Gatsby supports Node 6 and up, we introduce `npx` here as an optional command for users of npm 5.2.0.

Reviewer 2: ```suggestion
> npm 5.2.0--bundled with Node 8--introduced a command called `npx`. Gatsby supports Node 6 and up, we introduce `npx` here as an optional command for users of npm 5.2.0 or greater.
```

Something like this perhaps?

### Share best practices whenever possible

There is an infinite number of tasks that the documentation and tutorials could include, so they should not attempt to teach every way of doing a task. Usually, they should teach the following:

> - the most fundamental way possible to do a task with Gatsby

Author: Should we just commit to teach best practices? I think "unstructured data" is an example of a common, fundamental way of bringing data into a site that we wouldn't necessarily recommend as "best practice"; however, lots of people complained about having to learn GraphQL and wanted to know how to pull in data another (arguably more common) way.

Reviewer 2: This is such a tough question to answer/document because it's subjective and sorta changing 🙃

I like how you've structured this, however. I read this--intentional or not--as sort of an ordered preference, i.e. we should prefer to show the most fundamental way, then the most common, then the best practice after we've shown the fundamentals/common approaches.

Maybe that helps clarify this? We should document best practices whenever we can, but only when users have the necessary context.
